src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -331,7 +331,7 @@ internal override void RequestRun()
         {
             base.RequestRun();
 
-            if (RunSettings.RunType != RunType.Manual)
+            if (RunSettings.RunType != RunType.Manual && RunSettings.RunEnabled)
             {
```

Comment:
will feel a little better if put runenabled check first, as it is kind of precondition before run type..